projects/frontend/shared-components/gui/projects/shared/src/lib/features/placeholder/components/placeholder.component.ts

```diff
@@ -68,7 +68,7 @@ const ErrorEscalation = {
 
 const ErrorImgSource = {
     Generic: 'assets/images/placeholder/server-error.svg',
-    Offline: null,
+    Offline: '',
```

Comment:
I think yes, but I should check the code impl, it it's evaluating with negation or with utils checking for string, on a first call I would say better to left it like it was with `null` value

Reply:
It gets passed to this template.
https://github.com/vmware/versatile-data-kit/blob/main/projects/frontend/shared-components/gui/projects/shared/src/lib/features/placeholder/components/placeholder.component.html#L45

https://github.com/vmware/versatile-data-kit/blob/main/projects/frontend/shared-components/gui/projects/shared/src/lib/features/placeholder/components/placeholder.component.html#L56

And then the underlying function seems to do nothing with it.

https://github.com/vmware/versatile-data-kit/blob/main/projects/frontend/shared-components/gui/projects/shared/src/lib/features/placeholder/components/placeholder.component.ts#L542

I'm not super familiar with Angular. @gorankokin Please let me know if something is going on under the hood. 😕

Reply:
# After default error template is rendered `errorTemplateSystemDefault` https://github.com/vmware/versatile-data-kit/blob/main/projects/frontend/shared-components/gui/projects/shared/src/lib/features/placeholder/components/placeholder.component.html#L95
# in the template there is *ngIf statement what image to render https://github.com/vmware/versatile-data-kit/blob/main/projects/frontend/shared-components/gui/projects/shared/src/lib/features/placeholder/components/placeholder.component.html#L98
and from what I'm seeing it will work ok.